SITE_CONTRIBUTION.md

```diff
@@ -0,0 +1,134 @@
+# Website Contributions
+
+The projectcontour.io site is a static HTML site built using the Jekyll platform. Site content is written in markdown with liquid templating to populate content sections such as links.
+
+Contribution [guidelines](/CONTRIBUTING.md) and [workflows](/CONTRIBUTING.md#contribution-workflow) still apply to site contributions.
+
+Please familiarize yourself with Jekyll and Liquid templating if you intend on contributing to the site.
+
+- [Jekyll documentation][1]
+- [Liquid Templating][2]
+- [Mastering Markdown][5]
+  
+## Directory Structure
+
+Overview of the site directory structure and purpose of directories:
+
+```
+site
+├── _contributors
+├── css # Fonts
+│   └── fonts
+│       ├── IcoMoon
+│       └── Metropolis
+├── _data # Custom data files
+├── docs # Versioned product documentation
+│   ├── master
+│   └── v1.0.0
+├── _guides # Unversioned product usage guides
+├── img
+│   ├── case-study-icons
+│   ├── cert-manager
+│   ├── contour-1.0
+│   ├── contributors
+│   ├── heroes
+│   └── posts
+├── _includes # HTML partials that are incorporated into layouts
+├── js # Javascript files
+├── _layouts # Layout templates to wrap posts and pages
+├── _metrics
+├── _plugins # Ruby files for custom Jekyll plugins
+├── _posts # Contour blog posts
+├── _resources # General information pages
+└── _scss # Site style sheets
+    ├── bootstrap-4.1.3
+    │   ├── mixins
+    │   └── utilities
+    └── site
+        ├── common
+        ├── layouts
+        ├── objects
+        ├── settings
+        └── utilities
+```
+
+## Link Insertion Standards
+
+Usage of links within a page has been standardized to simplify the process of finding and fixing links as required.
+
+Avoid using inline links, instead, use a reference table the bottom of the file; making it easier to find and update links. [Reference Links][3]. There are some exceptions to this rule, such as depreciation warnings.
```

Comment:
Nit: Sadly, this should be `deprecation warnings`, not `depreciation warnings`. Thanks, English, for having such a large difference in meaning by one letter.